airflow/www/templates/airflow/dags.html

```diff
@@ -403,7 +403,13 @@ <h2>DAGs</h2>
       });
       d3.json("{{ url_for('Airflow.task_stats') }}", function(error, json) {
         for(var dag_id in json) {
-            states = json[dag_id];
+            states = json[dag_id].map(task => {
```

Comment:
Why not pass another value to d3 if it causes problems?
It seems much simpler to me

Line 366
```diff
-              .attr('title', function(d) {return d.state})
+              .attr('title', function(d) {return d.state || 'none' })
```

Reply:
Sounds reasonable. If we do want to keep the null state instead of none, your solution is better 👍

Reply:
I thought it might be the better decision to have a None state in js as we have in Python. But Python None probably should equal Null in JS.

And your change really only changes the displayed value which I think is probably more appropriate. So I will go for your solution.